src/app/shared/cms/components/cms-product-list-rest/cms-product-list-rest.component.spec.ts

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line ish-custom-rules/ban-imports-file-pattern
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CMSProductListRestComponent } from './cms-product-list-rest.component';
+
+describe('Cms Product List Rest Component', () => {
+  let component: CMSProductListRestComponent;
+  let fixture: ComponentFixture<CMSProductListRestComponent>;
+  let element: HTMLElement;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      declarations: [CMSProductListRestComponent],
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CMSProductListRestComponent);
+    component = fixture.componentInstance;
+    element = fixture.nativeElement;
+  });
+
+  it('should be created', () => {
+    expect(component).toBeTruthy();
+    expect(element).toBeTruthy();
+    expect(() => fixture.detectChanges()).not.toThrow();
+  });
```

Comment:
test for the getProductSku function is missing?

Reply:
added test that tests the mapping of response JSON with the mapper function and limits its displayed number.